op25/gr-op25_repeater/apps/multi_rx.py

```diff
@@ -774,7 +763,7 @@ def byteify(input):    # thx so
                 parser.print_help()
                 exit(1)
             config = json.loads(open(options.config_file).read())
-        self.tb = rx_block(options.verbosity, config = byteify(config))
```

Comment:
I'm not sure why this was ever here, to be honest. It seems to work fine without it on both Python 2 and 3. Am I missing something?

Reply:
Reviewing the code, I can't see a reason why it's necessary either. Perhaps a very old version of Python 2 needed it?